tests/e2e/core-tests/specs/merchant/wp-admin-product-new.test.js

```diff
@@ -84,12 +84,19 @@ const runAddSimpleProductTest = () => {
 
 const runAddVariableProductTest = () => {
 	describe('Add New Variable Product Page', () => {
-		it('can create product with variations', async () => {
+		beforeAll(async () => {
+			await merchant.login();
+		});
```

Comment:
I like this. I wonder if we also want to start thinking about adding in a logout step after all the tests run, e.g.

```
afterAll(async () => {
	await merchant.logout();
});
```

Just to prevent future flakiness and make the tests a bit more atomic and clean up after themselves. We don't need to change this here, but just a thought.

Reply:
Good suggestion. I created #29796.